src/hint_processor/builtin_hint_processor/secp/field_utils.rs

```diff
@@ -220,6 +221,29 @@ mod tests {
     #[test]
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
     fn run_verify_zero_ok() {
+        let hint_codes = vec![
+            &hint_code::VERIFY_ZERO_V1,
+            &hint_code::VERIFY_ZERO_V2,
+            &hint_code::VERIFY_ZERO_V3,
+        ];
+        for hint_code in hint_codes {
+            let mut vm = vm_with_range_check!();
+            //Initialize run_context
+            run_context!(vm, 0, 9, 9);
+            //Create hint data
+            let ids_data = non_continuous_ids_data![("val", -5), ("q", 0)];
+            vm.segments = segments![((1, 4), 0), ((1, 5), 0), ((1, 6), 0)];
+            //Execute the hint
+            assert!(run_hint!(vm, ids_data, hint_code, exec_scopes_ref!()).is_ok());
+            //Check hint memory inserts
+            //ids.q
+            check_memory![vm.segments.memory, ((1, 9), 0)];
+        }
+    }
```

Comment:
```suggestion
    #[rstest]
    #[case(hint_code::VERIFY_ZERO_V1)]
    #[case(hint_code::VERIFY_ZERO_V2)]
    #[case(hint_code::VERIFY_ZERO_V3)]
    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
    fn run_verify_zero_ok(hint_code: &str) {
        let mut vm = vm_with_range_check!();
        //Initialize run_context
        run_context!(vm, 0, 9, 9);
        //Create hint data
        let ids_data = non_continuous_ids_data![("val", -5), ("q", 0)];
        vm.segments = segments![((1, 4), 0), ((1, 5), 0), ((1, 6), 0)];
        //Execute the hint
        assert!(run_hint!(vm, ids_data, hint_code, exec_scopes_ref!()).is_ok());
        //Check hint memory inserts
        //ids.q
        check_memory![vm.segments.memory, ((1, 9), 0)];
    }
```

Reply:
I merged the hint-62 branch into this one because I needed it for the integration test to work, this code belongs to that branch and not to this PR, sorry for the inconvenience